crates/biome_markdown_syntax/src/generated/kind.rs

```diff
@@ -138,6 +169,11 @@ impl MarkdownSyntaxKind {
             WHITESPACE3 => "   ",
             UNDERSCORE => "_",
             HASH => "#",
+            PIPE => "|",
+            GREATER_THAN => ">",
+            PLUS => "+",
+            PERIOD => ".",
```

Comment:
When does the dot becomes a known token in markdown? I can't recall

Reply:
It is purely for list blocks I believe.

Reply:
Then let's call it DOT